### DISCUSSION TITLE
DatabricksWorkflowTaskGroup and templated task parameters

### DISCUSSION THREAD

**Discussion by avivshafir** [apache/airflow#42399] [Q&A]
```
    task_group = DatabricksWorkflowOperator(
        group_id="test",
        databricks_conn_id=dbx_conn_id,
        job_clusters=[
            {
                "job_cluster_key": "cpu_cluster",
                "new_cluster": {
                    "spark_version": "15.3.x-scala2.12",
                    "node_type_id": "cgd-fleet.xlarge",
                    "driver_node_type_id": "cgd-fleet.xlarge",
                    "num_workers": 1,
                },
            }
        ],
    )

    with task_group:
        a_task = DatabricksNotebookOperator(
            task_id="a_task",
            databricks_conn_id=dbx_conn_id,
            notebook_path="test/notebook_a",
            job_cluster_key="cpu_cluster",
            source="GIT",
            notebook_params={
                "company_db_table_name": "{{ var.json.table_name }}"
            },
        )
```

When I pass a templated variable to my notebook operator it seems it's being overridden by the workflow params
so in the execution in databricks I see the parameter as {{ var.json.table_name }}
but I want to use a templated variable, different arguments for different notebook,
I don't want to pass those at the workflow task group level which also doesn't render the templated variables.
what am I missing here? 

Probably related to what's written here https://airflow.apache.org/docs/apache-airflow-providers-databricks/stable/operators/workflow.html

> "To minimize update conflicts, we recommend that you keep parameters in the notebook_params of the DatabricksWorkflowTaskGroup and not in the DatabricksNotebookOperator whenever possible. This is because, tasks in the DatabricksWorkflowTaskGroup are passed in on the job trigger time and do not modify the job definition."

**Comment by anilreddaboina54** [comment]
Hi Team, 
I'm also seeing the similar kind of issue with DatabricksWorkflowTaskGroup, can you please assist on this.
